raco/backends/sql/catalog.py

```diff
@@ -107,10 +107,12 @@ def _convert_zeroary_expr(self, cols, expr, input_scheme):
 
     def _convert_unary_expr(self, cols, expr, input_scheme):
         input = self._convert_expr(cols, expr.input, input_scheme)
-        if isinstance(expr, expression.MAX):
-            return func.max(input)
-        if isinstance(expr, expression.MIN):
-            return func.min(input)
+
+        fname = expr.__class__.__name__.lower()
```

Comment:
This is certainly an elegant approach, but I think it doesn't work for at least half of the existing aggregates in raco. Specifically, of the unary aggregates in `raco/raco/expression/aggregate.py`, I cannot find a corresponding function for `COUNTALL`, `AVG`, `STDEV` in the list at http://docs.sqlalchemy.org/en/latest/core/functions.html. So I think some special-casing is unfortunately necessary. This is considerably mitigated by the fact that we can just say `func.foo(input)` for any name `foo` and SQLAlchemy will generate the corresponding SQL. So the special cases I think we need to handle right now (for Postgres compatibility) are:
```
AVG -> avg
COUNTALL -> count(*)
STDEV -> stddev_samp
```